MAUI/Cartesian-Charts/neumorphicUI-column-chart.md

```diff
@@ -0,0 +1,245 @@
+
+{% endtabs %}
+
+## Step 5: Implementing a SfNeumorphismView Class
+
+The `SfNeumorphismView` class extends `ContentView` and provides a container for displaying a neumorphic (soft shadow) effect. It includes a `Grid` and a `GraphicsView` for rendering the neumorphic shadow based on the provided `SfNeumorphismDrawer`. Additionally, it exposes properties and methods to manage the drawable content and handle property changes.
+
+{% tabs %}
+
+{% highlight SfNeumorphismView %}
+
+    public SfNeumorphismView()
+    {
+        Drawable = new SfNeumorphismDrawer();
+
+        grid = new Grid();
+        grid.Margin = new Thickness(0);
+
+        graphicsView = new GraphicsView();
+        graphicsView.Margin = new Thickness(0);
+        graphicsView.BackgroundColor = Colors.Transparent;
+        graphicsView.SetBinding(GraphicsView.DrawableProperty, new Binding() { Path = nameof(Drawable), Source = this });
+
+        grid.Children.Add(graphicsView);
+        base.Content = grid;
+    }
+    public void Invalidate()
+    {
+        graphicsView.Invalidate();
+    }
+
+{% endhighlight SfNeumorphismView %}
+
+{% endtabs %}
+
+## Step 6: Implementing the SfNeumorphismColumnSeries and SfNeumorphismColumnSegment Classes
+
+The `SfNeumorphismColumnSeries` class is a custom implementation of the `ColumnSeries` in the Syncfusion.Maui.Charts library. It introduces a neumorphic (soft shadow) effect to column series by associating a `SfNeumorphismDrawer` with each column segment.
+
+This class overrides the base method to create a custom `SfNeumorphismColumnSegment` instead of the default `ColumnSegment`. Associates the `Drawable` property with the newly created segment.
+
+{% tabs %}
+
+{% highlight SfNeumorphismColumnSeries %}
+
+    public class SfNeumorphismColumnSeries : ColumnSeries
+    {
+    protected override ChartSegment CreateSegment()
+    {
+        return new SfNeumorphismColumnSegment(Drawable);
+    }
+    }
+
+{% endhighlight SfNeumorphismColumnSeries %}
+
+{% endtabs %}
+
+
+The `SfNeumorphismColumnSegment` class is a custom implementation of the `ColumnSegment` in the Syncfusion.Maui.Charts library. It enhances the default column segment by incorporating a neumorphic shadow effect using the associated `SfNeumorphismDrawer`.
+
+It Overrides the base method to implement custom drawing logic. Draws the neumorphic shadow using the `SfNeumorphismDrawer` associated with the segment.
+
+{% tabs %}
+
+{% highlight SfNeumorphismColumnSegment %}
+
+    public class SfNeumorphismColumnSegment : ColumnSegment
+    {
+    protected override void Draw(ICanvas canvas)
+    {
+        if (Series is ColumnSeries series && series.ActualYAxis is NumericalAxis yAxis)
+        {
+            var top = yAxis.ValueToPoint(Convert.ToDouble(yAxis.Maximum ?? double.NaN));
+    
+            var trackRect = new RectF() { Left = Left, Top = top, Right = Right, Bottom = Bottom };
+    
+            Drawable.Draw(canvas, trackRect);
+        }
+    }
```

Comment:
@DhanarajRajendran 
Code alignment is not proper

Reply:
Aligned the code properly.